nss-day-cohort-39/tabloid-full-stack-coulees
# Date Picker for Post Add/Edit Forms

### Changes
1. Added reactstrap-date-picker to the edit post and add post forms

### Steps to Test
1. in you client directory with your package.json file run the command `npm install reactstrap-date-picker`
2. Test the edit form to see if the date picker populates with the publish date for posts that have publish date
3. Test, generally, edit and add post functions
